### PR TITLE
Introduce GetInstanceFull

### PR DESCRIPTION
**Pull request by stgraber**
Let's make use of the new `?recursion=1` that @presztak introduced recently and make some of our CLI tools significantly faster in the process :)

### CODE REVIEW

File: client/lxd_instances.go [hunk 1]
```diff
@@ -224,6 +224,54 @@ func (r *ProtocolLXD) GetInstance(name string) (*api.Instance, string, error) {
 	return &instance, etag, nil
 }
 
+// GetInstanceFull returns the instanceentry for the provided name along with snapshot information.
```
Comment (tomponline) [inline client/lxd_instances.go:227]: Typo here

**Review by tomponline — APPROVED** [review]
One typo in comment

**Comment by stgraber** [comment]
Fixed!

**Comment by tomponline** [comment]
Thanks

**Pull request merged**